src/styles/blocks/_main.scss

```diff
@@ -1,11 +1,8 @@
 @import "./utils/variables";
 
 .main {
-  max-width: #{$contentWidth};
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: space-between;
-  &__product-card {
-    margin-bottom: 50px;
-  }
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
```

Comment:
It's better to make auto-fill for seeing all content on smaller screens.

Reply:
Task didn't include adaptive layout for small screens, so it's fine